### PR TITLE
improvement: handle replacement tx

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

**Review by nguyenhoaidanh — APPROVED** [review]

File: src/state/transactions/hooks.tsx [hunk 1]
```diff
@@ -46,13 +49,20 @@ export function useTransactionAdder(): (
       if (!chainId) return
 
       const { hash } = response
+
+      const tx = await library?.getTransaction(hash)
```
Comment (namgold) [inline src/state/transactions/hooks.tsx:53]: Will this throw error, causing we cant dispatch `addTransaction`?

Comment (namgold) [reply src/state/transactions/hooks.tsx:53]: Should we wrap this into a try catch?

Comment (viet-nv) [reply src/state/transactions/hooks.tsx:53]: no need. It will return null if it's not existed

Comment (namgold) [reply src/state/transactions/hooks.tsx:53]: r u sure?

**Pull request merged**